src/MarginTrading.Backend.Services/AccountManager.cs

```diff
@@ -339,10 +350,18 @@ private string GetTradingConditions()
             }
         }
 
+        private async Task<MarginTradingAccount> CreateAccountAndWalletAsync(string clientId, string baseAssetId, string tradingConditionId)
+        {
```

Comment:
Should throw exception if server is not Live to avoid mistakes in usage. May be even better to name method like.. CreateLiveAccountAndWalletAsync
Or even better to remove this method at all and add "if" in CreateAccount method.
If server is live - create wallet and get Id, if demo, just generate new Id.
In that case is in no need to think what method we should call.